.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Quorum reporting docker push
```

Comment:
As the tag version follows standard `vX.Y.Z` pattern, you can simplify the workflow like:
https://github.com/ConsenSys/qubernetes/blob/master/.github/workflows/publish_docker_image.yaml